### PR TITLE
Use hello_world as benchmark

### PR DESCRIPTION
**Pull request by rbehjati**
Ref #1933

### CODE REVIEW

File: oak_functions/loader/src/tests.rs [hunk 1]
```diff
@@ -26,6 +26,8 @@ use std::net::{Ipv6Addr, SocketAddr};
 use test::Bencher;
 
 const TEST_WASM_MODULE_PATH: &str = "testdata/non-oak-minimal.wasm";
+const HELLO_WORLD_MODULE_PATH: &str = "testdata/hello_world.wasm";
```
Comment (rbehjati) [inline oak_functions/loader/src/tests.rs:29]: I could also use this in the `test_server` test. WDYT?

File: oak_functions/loader/src/tests.rs [hunk 2]
```diff
@@ -94,7 +96,11 @@ fn bench_wasm_handler(bencher: &mut Bencher) {
         // to be less than a fixed threshold.
         // Note: `summary.mean` is in nanoseconds, even though it is not explicitly documented in
         // https://doc.rust-lang.org/test/stats/struct.Summary.html.
-        assert!(summary.mean < 50_000.0, "elapsed time: {}ns", summary.mean);
+        assert!(
+            summary.mean < 1_000_000.0,
```
Comment (rbehjati) [inline oak_functions/loader/src/tests.rs:100]: Now it takes more than 600ns.

Comment (tiziano88) [reply oak_functions/loader/src/tests.rs:106]: :grimacing: why is that? There's not much happening, is it?

File: oak_functions/loader/src/tests.rs (hunk 1 above)
Comment (tiziano88) [reply oak_functions/loader/src/tests.rs:34]: Why does it get compiled to testdata? That seems odd, given it's not checked-in.

(edit): I just noticed it is checked in, that is not great, I think it should just compile it to some target / bin folder and pick it up from there, like we do for the rest of the examples.

File: oak_functions/loader/src/tests.rs (hunk 2 above)
Comment (rbehjati) [reply oak_functions/loader/src/tests.rs:106]: This one is actually reading the request and writing the response. I suppose that is taking some time. The whole thing is still less than 1ms. Is that bad?

Comment (tiziano88) [reply oak_functions/loader/src/tests.rs:106]: Ah you are right, I thought it was taking 1s now! Never mind, all good! :)

Comment (rbehjati) [reply oak_functions/loader/src/tests.rs:106]: Yeah! I made that mistake before too :D

File: oak_functions/loader/Cargo.toml [hunk 3]
```diff
@@ -34,6 +34,7 @@ wasmi = "*"
 [dev-dependencies]
 hyper = { version = "*", features = ["client"] }
 tempfile = "*"
+test_utils = { version = "0.1.0", path = "../sdk/test_utils" }
```
Comment (tiziano88) [inline oak_functions/loader/Cargo.toml:37]:
```suggestion
test_utils = { path = "../sdk/test_utils" }
```

File: oak_functions/examples/hello_world/module/src/tests.rs [hunk 4]
```diff
@@ -48,7 +48,7 @@ async fn start_client(port: u16, notify_sender: tokio::sync::oneshot::Sender<()>
     let request = hyper::Request::builder()
         .method(http::Method::POST)
         .uri(format!("http://localhost:{}/invoke", port))
-        .body(hyper::Body::from("Harry Potter"))
+        .body(hyper::Body::from("World"))
```
Comment (tiziano88) [inline oak_functions/examples/hello_world/module/src/tests.rs:51]: So serious now :)

**Review by tiziano88 — APPROVED** [review]

File: oak_functions/loader/src/tests.rs (hunk 1 above)
Comment (rbehjati) [reply oak_functions/loader/src/tests.rs]: Thanks. Done.

File: oak_functions/examples/hello_world/module/src/tests.rs (hunk 4 above)
Comment (rbehjati) [reply oak_functions/examples/hello_world/module/src/tests.rs:51]: Indeed :D

**Pull request merged**